aiinfra-cluster/main.tf

```diff
@@ -188,4 +188,5 @@ module "aiinfra-default-dashboard" {
   base_dashboard  = "Empty"
   title           = "AI Accelerator Experience Dashboard"
   widgets         = local.widgets
+  depends_on      = [module.aiinfra-compute]
```

Comment:
We should skip creating the dashboard if Ops agent is disabled.

Reply:
updated to depend on the dashboard module which has the metric descriptor resources